pypeit/pypeit.py

```diff
@@ -279,6 +280,7 @@ def calib_all(self):
                     self.calibrations_path, qadir=self.qa_path, reuse_masters=self.reuse_masters,
                     show=self.show, slitspat_num=self.par['rdx']['slitspatnum'])
                 # Do it
+                # TODO: Why isn't set_config part of the Calibrations.__init__ method?
```

Comment:
There was a time when we instantiated once and then
reused the class, i.e. reset its configuration.  

If that is no longer the case then we
can have the config be embedded.